02:56:38 p. m.: DEBUG [connectionpool.py:1001] Starting new HTTPS connection (1): datos.cultura.gob.ar:443
02:56:40 p. m.: DEBUG [connectionpool.py:456] https://datos.cultura.gob.ar:443 "GET /dataset/37305de4-3cce-4d4b-9d9a-fec3ca61d09f/resource/4207def0-2ff7-41d5-9095-d42ae8207a5d/download/museos_datosabiertos.csv HTTP/1.1" 200 304431
02:57:36 p. m.: DEBUG [connectionpool.py:1001] Starting new HTTPS connection (1): datos.cultura.gob.ar:443
02:57:37 p. m.: DEBUG [connectionpool.py:456] https://datos.cultura.gob.ar:443 "GET /dataset/37305de4-3cce-4d4b-9d9a-fec3ca61d09f/resource/4207def0-2ff7-41d5-9095-d42ae8207a5d/download/museos_datosabiertos.csv HTTP/1.1" 200 304431
03:34:44 p. m.: DEBUG [connectionpool.py:1001] Starting new HTTPS connection (1): datos.cultura.gob.ar:443
03:34:45 p. m.: DEBUG [connectionpool.py:456] https://datos.cultura.gob.ar:443 "GET /dataset/37305de4-3cce-4d4b-9d9a-fec3ca61d09f/resource/4207def0-2ff7-41d5-9095-d42ae8207a5d/download/museos_datosabiertos.csv HTTP/1.1" 200 304431
03:35:12 p. m.: DEBUG [connectionpool.py:1001] Starting new HTTPS connection (1): datos.cultura.gob.ar:443
03:35:13 p. m.: DEBUG [connectionpool.py:456] https://datos.cultura.gob.ar:443 "GET /dataset/37305de4-3cce-4d4b-9d9a-fec3ca61d09f/resource/4207def0-2ff7-41d5-9095-d42ae8207a5d/download/museos_datosabiertos.csv HTTP/1.1" 200 304431
03:35:15 p. m.: DEBUG [connectionpool.py:456] https://datos.cultura.gob.ar:443 "GET /dataset/37305de4-3cce-4d4b-9d9a-fec3ca61d09f/resource/392ce1a8-ef11-4776-b280-6f1c7fae16ae/download/cine.csv HTTP/1.1" 200 96427
03:35:16 p. m.: DEBUG [connectionpool.py:456] https://datos.cultura.gob.ar:443 "GET /dataset/37305de4-3cce-4d4b-9d9a-fec3ca61d09f/resource/01c6c048-dbeb-44e0-8efa-6944f73715d7/download/biblioteca_popular.csv HTTP/1.1" 200 511527
03:57:52 p. m.: INFO [main.py:25] Ejecución ArchivosCsv.descarga_archivos()
03:57:54 p. m.: INFO [main.py:37] Comienza descarga de museos
03:57:54 p. m.: INFO [main.py:41] Ocurrió un error Cannot save file into a non-existent directory: 'D:\Lautaro\Python\challenge-data-alkemy\data\2022-abril\museos', reintentando descarga
03:57:54 p. m.: INFO [main.py:44] Archivo museos descargado con éxito
03:57:56 p. m.: INFO [main.py:37] Comienza descarga de salas-de-cine
03:57:56 p. m.: INFO [main.py:41] Ocurrió un error Cannot save file into a non-existent directory: 'D:\Lautaro\Python\challenge-data-alkemy\data\2022-abril\salas-de-cine', reintentando descarga
03:57:56 p. m.: INFO [main.py:44] Archivo salas-de-cine descargado con éxito
03:57:57 p. m.: INFO [main.py:37] Comienza descarga de bibliotecas
03:57:57 p. m.: INFO [main.py:41] Ocurrió un error Cannot save file into a non-existent directory: 'D:\Lautaro\Python\challenge-data-alkemy\data\2022-abril\bibliotecas', reintentando descarga
03:57:57 p. m.: INFO [main.py:44] Archivo bibliotecas descargado con éxito
03:58:06 p. m.: INFO [main.py:25] Ejecución ArchivosCsv.descarga_archivos()
03:58:08 p. m.: INFO [main.py:37] Comienza descarga de museos
03:58:08 p. m.: INFO [main.py:39] Archivo museos descargado con éxito
03:58:09 p. m.: INFO [main.py:37] Comienza descarga de salas-de-cine
03:58:09 p. m.: INFO [main.py:39] Archivo salas-de-cine descargado con éxito
03:58:11 p. m.: INFO [main.py:37] Comienza descarga de bibliotecas
03:58:11 p. m.: INFO [main.py:39] Archivo bibliotecas descargado con éxito
03:58:27 p. m.: INFO [main.py:25] Ejecución ArchivosCsv.descarga_archivos()
03:58:29 p. m.: INFO [main.py:37] Comienza descarga de museos
03:58:29 p. m.: INFO [main.py:39] Archivo museos descargado con éxito
03:58:30 p. m.: INFO [main.py:37] Comienza descarga de salas-de-cine
03:58:30 p. m.: INFO [main.py:39] Archivo salas-de-cine descargado con éxito
03:58:31 p. m.: INFO [main.py:37] Comienza descarga de bibliotecas
03:58:31 p. m.: INFO [main.py:39] Archivo bibliotecas descargado con éxito
04:03:33 p. m.: INFO [main.py:36] Ejecución ArchivosCsv.descarga_archivos()
04:03:57 p. m.: INFO [main.py:36] Ejecución ArchivosCsv.descarga_archivos()
04:03:59 p. m.: INFO [main.py:47] Comienza descarga de museos
04:03:59 p. m.: INFO [main.py:49] Archivo museos descargado con éxito
04:04:00 p. m.: INFO [main.py:47] Comienza descarga de salas-de-cine
04:04:00 p. m.: INFO [main.py:49] Archivo salas-de-cine descargado con éxito
04:04:02 p. m.: INFO [main.py:47] Comienza descarga de bibliotecas
04:04:02 p. m.: INFO [main.py:49] Archivo bibliotecas descargado con éxito
11:47:29 a. m.: INFO [main.py:107] Challenge de Data Analytics - Alkemy
11:47:29 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:47:29 a. m.: INFO [main.py:35] Se crean los directorios. Continúa el proceso de descarga.
11:47:32 a. m.: INFO [main.py:85] Comienza descarga de museos
11:47:32 a. m.: INFO [main.py:87] Archivo museos descargado con éxito
11:47:32 a. m.: INFO [main.py:97] Se transforma data de museos
11:48:02 a. m.: INFO [main.py:108] Challenge de Data Analytics - Alkemy
11:48:02 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:48:02 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:48:04 a. m.: INFO [main.py:86] Comienza descarga de museos
11:48:04 a. m.: INFO [main.py:88] Archivo museos descargado con éxito
11:48:04 a. m.: INFO [main.py:98] Se transforma data de museos
11:48:43 a. m.: INFO [main.py:109] Challenge de Data Analytics - Alkemy
11:48:43 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:48:43 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:48:46 a. m.: INFO [main.py:87] Comienza descarga de museos
11:48:46 a. m.: INFO [main.py:89] Archivo museos descargado con éxito
11:48:46 a. m.: INFO [main.py:99] Se transforma data de museos
11:48:54 a. m.: INFO [main.py:109] Challenge de Data Analytics - Alkemy
11:48:54 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:48:54 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:48:56 a. m.: INFO [main.py:87] Comienza descarga de museos
11:48:56 a. m.: INFO [main.py:89] Archivo museos descargado con éxito
11:48:56 a. m.: INFO [main.py:99] Se transforma data de museos
11:51:12 a. m.: INFO [main.py:110] Challenge de Data Analytics - Alkemy
11:51:12 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:51:12 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:51:15 a. m.: INFO [main.py:88] Comienza descarga de museos
11:51:15 a. m.: INFO [main.py:90] Archivo museos descargado con éxito
11:51:15 a. m.: INFO [main.py:100] Se transforma data de museos
11:51:25 a. m.: INFO [main.py:110] Challenge de Data Analytics - Alkemy
11:51:25 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:51:25 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:51:27 a. m.: INFO [main.py:88] Comienza descarga de museos
11:51:27 a. m.: INFO [main.py:90] Archivo museos descargado con éxito
11:51:27 a. m.: INFO [main.py:100] Se transforma data de museos
11:51:38 a. m.: INFO [main.py:110] Challenge de Data Analytics - Alkemy
11:51:38 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:51:38 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:51:40 a. m.: INFO [main.py:88] Comienza descarga de museos
11:51:40 a. m.: INFO [main.py:90] Archivo museos descargado con éxito
11:51:40 a. m.: INFO [main.py:100] Se transforma data de museos
11:52:47 a. m.: INFO [main.py:110] Challenge de Data Analytics - Alkemy
11:52:47 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:52:47 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:52:49 a. m.: INFO [main.py:88] Comienza descarga de museos
11:52:49 a. m.: INFO [main.py:90] Archivo museos descargado con éxito
11:52:49 a. m.: INFO [main.py:100] Se transforma data de museos
11:52:49 a. m.: INFO [main.py:35] Se crean los directorios. Continúa el proceso de descarga.
11:52:51 a. m.: INFO [main.py:88] Comienza descarga de salas-de-cine
11:52:51 a. m.: INFO [main.py:90] Archivo salas-de-cine descargado con éxito
11:52:51 a. m.: INFO [main.py:103] Se transforma data de salas-de-cine
11:52:51 a. m.: INFO [main.py:35] Se crean los directorios. Continúa el proceso de descarga.
11:52:52 a. m.: INFO [main.py:88] Comienza descarga de bibliotecas
11:52:52 a. m.: INFO [main.py:90] Archivo bibliotecas descargado con éxito
11:52:52 a. m.: INFO [main.py:103] Se transforma data de bibliotecas
11:54:14 a. m.: INFO [main.py:111] Challenge de Data Analytics - Alkemy
11:54:14 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:54:14 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:54:16 a. m.: INFO [main.py:89] Comienza descarga de museos
11:54:16 a. m.: INFO [main.py:91] Archivo museos descargado con éxito
11:54:16 a. m.: INFO [main.py:101] Se transforma data de museos
11:54:16 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:54:18 a. m.: INFO [main.py:89] Comienza descarga de salas-de-cine
11:54:18 a. m.: INFO [main.py:91] Archivo salas-de-cine descargado con éxito
11:54:18 a. m.: INFO [main.py:104] Se transforma data de salas-de-cine
11:54:18 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:54:19 a. m.: INFO [main.py:89] Comienza descarga de bibliotecas
11:54:19 a. m.: INFO [main.py:91] Archivo bibliotecas descargado con éxito
11:54:19 a. m.: INFO [main.py:104] Se transforma data de bibliotecas
11:54:56 a. m.: INFO [main.py:111] Challenge de Data Analytics - Alkemy
11:54:56 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:54:56 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:55:17 a. m.: INFO [main.py:111] Challenge de Data Analytics - Alkemy
11:55:17 a. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
11:55:17 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:55:19 a. m.: INFO [main.py:89] Comienza descarga de museos
11:55:19 a. m.: INFO [main.py:91] Archivo museos descargado con éxito
11:55:19 a. m.: INFO [main.py:101] Se transforma data de museos
11:55:19 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:55:20 a. m.: INFO [main.py:89] Comienza descarga de salas-de-cine
11:55:20 a. m.: INFO [main.py:91] Archivo salas-de-cine descargado con éxito
11:55:20 a. m.: INFO [main.py:104] Se transforma data de salas-de-cine
11:55:20 a. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
11:55:21 a. m.: INFO [main.py:89] Comienza descarga de bibliotecas
11:55:21 a. m.: INFO [main.py:91] Archivo bibliotecas descargado con éxito
11:55:21 a. m.: INFO [main.py:104] Se transforma data de bibliotecas
12:02:59 p. m.: INFO [main.py:113] Challenge de Data Analytics - Alkemy
12:02:59 p. m.: INFO [main.py:74] Ejecución ArchivosCsv.descarga_archivos()
12:02:59 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:03:02 p. m.: INFO [main.py:89] Comienza descarga de museos
12:03:02 p. m.: INFO [main.py:92] Archivo museos descargado con éxito
12:03:02 p. m.: INFO [main.py:103] Se transforma data de museos
12:03:02 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:03:03 p. m.: INFO [main.py:89] Comienza descarga de salas-de-cine
12:03:03 p. m.: INFO [main.py:92] Archivo salas-de-cine descargado con éxito
12:03:03 p. m.: INFO [main.py:106] Se transforma data de salas-de-cine
12:03:03 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:03:05 p. m.: INFO [main.py:89] Comienza descarga de bibliotecas
12:03:05 p. m.: INFO [main.py:92] Archivo bibliotecas descargado con éxito
12:03:05 p. m.: INFO [main.py:106] Se transforma data de bibliotecas
12:13:30 p. m.: INFO [main.py:116] Challenge de Data Analytics - Alkemy
12:13:30 p. m.: INFO [main.py:77] Ejecución ArchivosCsv.descarga_archivos()
12:13:30 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:13:32 p. m.: INFO [main.py:92] Comienza descarga de museos
12:13:32 p. m.: INFO [main.py:95] Archivo museos descargado con éxito
12:13:32 p. m.: INFO [main.py:106] Se transforma data de museos
12:13:32 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:13:33 p. m.: INFO [main.py:92] Comienza descarga de salas-de-cine
12:13:33 p. m.: INFO [main.py:95] Archivo salas-de-cine descargado con éxito
12:13:33 p. m.: INFO [main.py:109] Se transforma data de salas-de-cine
12:16:00 p. m.: INFO [main.py:116] Challenge de Data Analytics - Alkemy
12:16:00 p. m.: INFO [main.py:77] Ejecución ArchivosCsv.descarga_archivos()
12:16:00 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:16:02 p. m.: INFO [main.py:92] Comienza descarga de museos
12:16:02 p. m.: INFO [main.py:95] Archivo museos descargado con éxito
12:16:02 p. m.: INFO [main.py:106] Se transforma data de museos
12:16:02 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:16:04 p. m.: INFO [main.py:92] Comienza descarga de salas-de-cine
12:16:04 p. m.: INFO [main.py:95] Archivo salas-de-cine descargado con éxito
12:16:04 p. m.: INFO [main.py:109] Se transforma data de salas-de-cine
12:16:04 p. m.: INFO [main.py:38] Directorios ya existentes. Continúa el proceso de descarga.
12:16:06 p. m.: INFO [main.py:92] Comienza descarga de bibliotecas
12:16:06 p. m.: INFO [main.py:95] Archivo bibliotecas descargado con éxito
12:16:06 p. m.: INFO [main.py:109] Se transforma data de bibliotecas
12:17:36 p. m.: INFO [main.py:115] Challenge de Data Analytics - Alkemy
12:17:36 p. m.: INFO [main.py:76] Ejecución ArchivosCsv.descarga_archivos()
12:17:36 p. m.: INFO [main.py:34] Se crean los directorios. Continúa el proceso de descarga.
12:17:38 p. m.: INFO [main.py:91] Comienza descarga de museos
12:17:38 p. m.: INFO [main.py:94] Archivo museos descargado con éxito
12:17:38 p. m.: INFO [main.py:105] Se transforma data de museos
12:17:38 p. m.: INFO [main.py:34] Se crean los directorios. Continúa el proceso de descarga.
12:17:39 p. m.: INFO [main.py:91] Comienza descarga de salas-de-cine
12:17:39 p. m.: INFO [main.py:94] Archivo salas-de-cine descargado con éxito
12:17:39 p. m.: INFO [main.py:108] Se transforma data de salas-de-cine
12:17:39 p. m.: INFO [main.py:34] Se crean los directorios. Continúa el proceso de descarga.
12:17:41 p. m.: INFO [main.py:91] Comienza descarga de bibliotecas
12:17:41 p. m.: INFO [main.py:94] Archivo bibliotecas descargado con éxito
12:17:41 p. m.: INFO [main.py:108] Se transforma data de bibliotecas
12:18:08 p. m.: INFO [main.py:114] Challenge de Data Analytics - Alkemy
12:18:08 p. m.: INFO [main.py:77] Ejecución ArchivosCsv.descarga_archivos()
12:18:08 p. m.: INFO [main.py:37] Directorios ya existentes. Continúa el proceso de descarga.
12:18:10 p. m.: INFO [main.py:90] Comienza descarga de museos
12:18:10 p. m.: INFO [main.py:93] Archivo museos descargado con éxito
12:18:10 p. m.: INFO [main.py:104] Se transforma data de museos
12:18:11 p. m.: INFO [main.py:37] Directorios ya existentes. Continúa el proceso de descarga.
12:18:12 p. m.: INFO [main.py:90] Comienza descarga de salas-de-cine
12:18:12 p. m.: INFO [main.py:93] Archivo salas-de-cine descargado con éxito
12:18:12 p. m.: INFO [main.py:107] Se transforma data de salas-de-cine
12:18:12 p. m.: INFO [main.py:37] Directorios ya existentes. Continúa el proceso de descarga.
12:18:14 p. m.: INFO [main.py:90] Comienza descarga de bibliotecas
12:18:14 p. m.: INFO [main.py:93] Archivo bibliotecas descargado con éxito
12:18:14 p. m.: INFO [main.py:107] Se transforma data de bibliotecas
12:20:46 p. m.: INFO [main.py:120] Challenge de Data Analytics - Alkemy
12:20:46 p. m.: INFO [main.py:81] Ejecución ArchivosCsv.descarga_archivos()
12:20:46 p. m.: INFO [main.py:39] Directorios ya existentes. Continúa el proceso de descarga.
12:20:49 p. m.: INFO [main.py:94] Comienza descarga de museos
12:20:49 p. m.: INFO [main.py:97] Archivo museos descargado con éxito
12:20:49 p. m.: INFO [main.py:108] Se transforma data de museos
12:20:49 p. m.: INFO [main.py:39] Directorios ya existentes. Continúa el proceso de descarga.
12:20:50 p. m.: INFO [main.py:94] Comienza descarga de salas-de-cine
12:20:50 p. m.: INFO [main.py:97] Archivo salas-de-cine descargado con éxito
12:20:50 p. m.: INFO [main.py:111] Se transforma data de salas-de-cine
12:20:50 p. m.: INFO [main.py:39] Directorios ya existentes. Continúa el proceso de descarga.
12:20:52 p. m.: INFO [main.py:94] Comienza descarga de bibliotecas
12:20:52 p. m.: INFO [main.py:97] Archivo bibliotecas descargado con éxito
12:20:52 p. m.: INFO [main.py:111] Se transforma data de bibliotecas
08:49:50 a. m.: INFO [main.py:156] Challenge de Data Analytics - Alkemy
08:49:50 a. m.: INFO [main.py:118] Ejecución ArchivosCsv.descarga_archivos()
08:49:50 a. m.: INFO [main.py:67] Se crean los directorios. Continúa el proceso de descarga.
08:49:52 a. m.: INFO [main.py:130] Comienza descarga de museos
08:49:52 a. m.: INFO [main.py:133] Archivo museos descargado con éxito
08:49:52 a. m.: INFO [main.py:144] Se transforma data de museos
08:49:52 a. m.: INFO [main.py:67] Se crean los directorios. Continúa el proceso de descarga.
08:49:54 a. m.: INFO [main.py:130] Comienza descarga de salas-de-cine
08:49:54 a. m.: INFO [main.py:133] Archivo salas-de-cine descargado con éxito
08:49:54 a. m.: INFO [main.py:147] Se transforma data de salas-de-cine
08:49:54 a. m.: INFO [main.py:67] Se crean los directorios. Continúa el proceso de descarga.
08:49:56 a. m.: INFO [main.py:130] Comienza descarga de bibliotecas
08:49:56 a. m.: INFO [main.py:133] Archivo bibliotecas descargado con éxito
08:49:56 a. m.: INFO [main.py:147] Se transforma data de bibliotecas
10:20:32 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
10:20:32 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
10:21:11 a. m.: INFO [postgres_client.py:40] Conectado con éxito a data_analytics
10:21:11 a. m.: INFO [postgres_client.py:41] Engine creado con éxito
10:36:51 a. m.: INFO [main.py:150] Challenge de Data Analytics - Alkemy
10:36:51 a. m.: INFO [main.py:112] Ejecución ArchivosCsv.descarga_archivos()
10:36:51 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:36:53 a. m.: INFO [main.py:124] Comienza descarga de museos
10:36:53 a. m.: INFO [main.py:127] Archivo museos descargado con éxito
10:36:53 a. m.: INFO [main.py:138] Se transforma data de museos
10:36:53 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:36:54 a. m.: INFO [main.py:124] Comienza descarga de salas-de-cine
10:36:54 a. m.: INFO [main.py:127] Archivo salas-de-cine descargado con éxito
10:36:54 a. m.: INFO [main.py:141] Se transforma data de salas-de-cine
10:37:36 a. m.: INFO [main.py:150] Challenge de Data Analytics - Alkemy
10:37:36 a. m.: INFO [main.py:112] Ejecución ArchivosCsv.descarga_archivos()
10:37:36 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:37:38 a. m.: INFO [main.py:124] Comienza descarga de museos
10:37:38 a. m.: INFO [main.py:127] Archivo museos descargado con éxito
10:37:38 a. m.: INFO [main.py:138] Se transforma data de museos
10:37:38 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:37:39 a. m.: INFO [main.py:124] Comienza descarga de salas-de-cine
10:37:39 a. m.: INFO [main.py:127] Archivo salas-de-cine descargado con éxito
10:37:39 a. m.: INFO [main.py:141] Se transforma data de salas-de-cine
10:37:39 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:37:41 a. m.: INFO [main.py:124] Comienza descarga de bibliotecas
10:37:41 a. m.: INFO [main.py:127] Archivo bibliotecas descargado con éxito
10:37:41 a. m.: INFO [main.py:141] Se transforma data de bibliotecas
10:38:26 a. m.: INFO [main.py:150] Challenge de Data Analytics - Alkemy
10:38:26 a. m.: INFO [main.py:112] Ejecución ArchivosCsv.descarga_archivos()
10:38:26 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:38:28 a. m.: INFO [main.py:124] Comienza descarga de museos
10:38:28 a. m.: INFO [main.py:127] Archivo museos descargado con éxito
10:38:28 a. m.: INFO [main.py:138] Se transforma data de museos
10:38:28 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:38:29 a. m.: INFO [main.py:124] Comienza descarga de salas-de-cine
10:38:29 a. m.: INFO [main.py:127] Archivo salas-de-cine descargado con éxito
10:38:29 a. m.: INFO [main.py:141] Se transforma data de salas-de-cine
10:38:29 a. m.: INFO [main.py:59] Directorios ya existentes. Continúa el proceso de descarga.
10:38:30 a. m.: INFO [main.py:124] Comienza descarga de bibliotecas
10:38:30 a. m.: INFO [main.py:127] Archivo bibliotecas descargado con éxito
10:38:30 a. m.: INFO [main.py:141] Se transforma data de bibliotecas
10:41:17 a. m.: INFO [postgres_client.py:40] Conectado con éxito a data_analytics
10:41:17 a. m.: INFO [postgres_client.py:41] Engine creado con éxito
10:57:46 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
10:57:46 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:01:03 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:01:03 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:01:03 a. m.: INFO [main.py:160] Challenge de Data Analytics - Alkemy
11:01:42 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:01:42 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:01:42 a. m.: INFO [main.py:160] Challenge de Data Analytics - Alkemy
11:01:42 a. m.: INFO [main.py:118] Ejecución ArchivosCsv.descarga_archivos()
11:01:42 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:01:44 a. m.: INFO [main.py:130] Comienza descarga de museos
11:01:44 a. m.: INFO [main.py:133] Archivo museos descargado con éxito
11:01:44 a. m.: INFO [main.py:144] Se transforma data de museos
11:01:44 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:01:46 a. m.: INFO [main.py:130] Comienza descarga de salas-de-cine
11:01:46 a. m.: INFO [main.py:133] Archivo salas-de-cine descargado con éxito
11:01:46 a. m.: INFO [main.py:147] Se transforma data de salas-de-cine
11:01:58 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:01:58 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:01:58 a. m.: INFO [main.py:160] Challenge de Data Analytics - Alkemy
11:01:58 a. m.: INFO [main.py:118] Ejecución ArchivosCsv.descarga_archivos()
11:01:58 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:02:00 a. m.: INFO [main.py:130] Comienza descarga de museos
11:02:00 a. m.: INFO [main.py:133] Archivo museos descargado con éxito
11:02:00 a. m.: INFO [main.py:144] Se transforma data de museos
11:02:00 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:02:01 a. m.: INFO [main.py:130] Comienza descarga de salas-de-cine
11:02:01 a. m.: INFO [main.py:133] Archivo salas-de-cine descargado con éxito
11:02:01 a. m.: INFO [main.py:147] Se transforma data de salas-de-cine
11:02:01 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:02:03 a. m.: INFO [main.py:130] Comienza descarga de bibliotecas
11:02:03 a. m.: INFO [main.py:133] Archivo bibliotecas descargado con éxito
11:02:03 a. m.: INFO [main.py:147] Se transforma data de bibliotecas
11:04:49 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:04:49 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:04:49 a. m.: INFO [main.py:165] Challenge de Data Analytics - Alkemy
11:04:49 a. m.: INFO [main.py:123] Ejecución ArchivosCsv.descarga_archivos()
11:04:49 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:04:52 a. m.: INFO [main.py:135] Comienza descarga de museos
11:04:52 a. m.: INFO [main.py:138] Archivo museos descargado con éxito
11:04:52 a. m.: INFO [main.py:149] Se transforma data de museos
11:04:52 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:04:53 a. m.: INFO [main.py:135] Comienza descarga de salas-de-cine
11:04:53 a. m.: INFO [main.py:138] Archivo salas-de-cine descargado con éxito
11:04:53 a. m.: INFO [main.py:152] Se transforma data de salas-de-cine
11:05:23 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:05:23 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:05:23 a. m.: INFO [main.py:165] Challenge de Data Analytics - Alkemy
11:05:23 a. m.: INFO [main.py:123] Ejecución ArchivosCsv.descarga_archivos()
11:05:23 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:05:25 a. m.: INFO [main.py:135] Comienza descarga de museos
11:05:25 a. m.: INFO [main.py:138] Archivo museos descargado con éxito
11:05:25 a. m.: INFO [main.py:149] Se transforma data de museos
11:05:25 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:05:27 a. m.: INFO [main.py:135] Comienza descarga de salas-de-cine
11:05:27 a. m.: INFO [main.py:138] Archivo salas-de-cine descargado con éxito
11:05:27 a. m.: INFO [main.py:152] Se transforma data de salas-de-cine
11:05:27 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:05:28 a. m.: INFO [main.py:135] Comienza descarga de bibliotecas
11:05:28 a. m.: INFO [main.py:138] Archivo bibliotecas descargado con éxito
11:05:28 a. m.: INFO [main.py:152] Se transforma data de bibliotecas
11:07:17 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:07:17 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:07:17 a. m.: INFO [main.py:168] Challenge de Data Analytics - Alkemy
11:07:17 a. m.: INFO [main.py:126] Ejecución ArchivosCsv.descarga_archivos()
11:07:17 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:07:20 a. m.: INFO [main.py:138] Comienza descarga de museos
11:07:20 a. m.: INFO [main.py:141] Archivo museos descargado con éxito
11:07:20 a. m.: INFO [main.py:152] Se transforma data de museos
11:07:20 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:07:21 a. m.: INFO [main.py:138] Comienza descarga de salas-de-cine
11:07:21 a. m.: INFO [main.py:141] Archivo salas-de-cine descargado con éxito
11:07:21 a. m.: INFO [main.py:155] Se transforma data de salas-de-cine
11:07:21 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:07:23 a. m.: INFO [main.py:138] Comienza descarga de bibliotecas
11:07:23 a. m.: INFO [main.py:141] Archivo bibliotecas descargado con éxito
11:07:23 a. m.: INFO [main.py:155] Se transforma data de bibliotecas
11:08:11 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:08:11 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:08:11 a. m.: INFO [main.py:168] Challenge de Data Analytics - Alkemy
11:08:11 a. m.: INFO [main.py:126] Ejecución ArchivosCsv.descarga_archivos()
11:08:11 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:08:13 a. m.: INFO [main.py:138] Comienza descarga de museos
11:08:13 a. m.: INFO [main.py:141] Archivo museos descargado con éxito
11:08:13 a. m.: INFO [main.py:152] Se transforma data de museos
11:08:13 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:08:14 a. m.: INFO [main.py:138] Comienza descarga de salas-de-cine
11:08:14 a. m.: INFO [main.py:141] Archivo salas-de-cine descargado con éxito
11:08:14 a. m.: INFO [main.py:155] Se transforma data de salas-de-cine
11:08:14 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:08:16 a. m.: INFO [main.py:138] Comienza descarga de bibliotecas
11:08:16 a. m.: INFO [main.py:141] Archivo bibliotecas descargado con éxito
11:08:16 a. m.: INFO [main.py:155] Se transforma data de bibliotecas
11:11:38 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:11:38 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:11:38 a. m.: INFO [main.py:168] Challenge de Data Analytics - Alkemy
11:11:38 a. m.: INFO [main.py:126] Ejecución ArchivosCsv.descarga_archivos()
11:11:38 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:11:40 a. m.: INFO [main.py:138] Comienza descarga de museos
11:11:40 a. m.: INFO [main.py:141] Archivo museos descargado con éxito
11:11:40 a. m.: INFO [main.py:152] Se transforma data de museos
11:11:40 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:11:42 a. m.: INFO [main.py:138] Comienza descarga de salas-de-cine
11:11:42 a. m.: INFO [main.py:141] Archivo salas-de-cine descargado con éxito
11:11:42 a. m.: INFO [main.py:155] Se transforma data de salas-de-cine
11:11:42 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:11:44 a. m.: INFO [main.py:138] Comienza descarga de bibliotecas
11:11:44 a. m.: INFO [main.py:141] Archivo bibliotecas descargado con éxito
11:11:44 a. m.: INFO [main.py:155] Se transforma data de bibliotecas
11:13:51 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:13:51 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:13:51 a. m.: INFO [main.py:171] Challenge de Data Analytics - Alkemy
11:13:51 a. m.: INFO [main.py:129] Ejecución ArchivosCsv.descarga_archivos()
11:13:51 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:13:54 a. m.: INFO [main.py:141] Comienza descarga de museos
11:13:54 a. m.: INFO [main.py:144] Archivo museos descargado con éxito
11:13:54 a. m.: INFO [main.py:155] Se transforma data de museos
11:13:54 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:13:55 a. m.: INFO [main.py:141] Comienza descarga de salas-de-cine
11:13:55 a. m.: INFO [main.py:144] Archivo salas-de-cine descargado con éxito
11:13:55 a. m.: INFO [main.py:158] Se transforma data de salas-de-cine
11:13:55 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:13:57 a. m.: INFO [main.py:141] Comienza descarga de bibliotecas
11:13:57 a. m.: INFO [main.py:144] Archivo bibliotecas descargado con éxito
11:13:57 a. m.: INFO [main.py:158] Se transforma data de bibliotecas
11:15:32 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:15:32 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:15:32 a. m.: INFO [main.py:171] Challenge de Data Analytics - Alkemy
11:15:32 a. m.: INFO [main.py:129] Ejecución ArchivosCsv.descarga_archivos()
11:15:32 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:15:34 a. m.: INFO [main.py:141] Comienza descarga de museos
11:15:34 a. m.: INFO [main.py:144] Archivo museos descargado con éxito
11:15:34 a. m.: INFO [main.py:155] Se transforma data de museos
11:15:34 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:15:35 a. m.: INFO [main.py:141] Comienza descarga de salas-de-cine
11:15:35 a. m.: INFO [main.py:144] Archivo salas-de-cine descargado con éxito
11:15:35 a. m.: INFO [main.py:158] Se transforma data de salas-de-cine
11:15:36 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:15:37 a. m.: INFO [main.py:141] Comienza descarga de bibliotecas
11:15:37 a. m.: INFO [main.py:144] Archivo bibliotecas descargado con éxito
11:15:37 a. m.: INFO [main.py:158] Se transforma data de bibliotecas
11:16:44 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:16:44 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:16:44 a. m.: INFO [main.py:171] Challenge de Data Analytics - Alkemy
11:16:44 a. m.: INFO [main.py:129] Ejecución ArchivosCsv.descarga_archivos()
11:16:44 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:16:46 a. m.: INFO [main.py:141] Comienza descarga de museos
11:16:46 a. m.: INFO [main.py:144] Archivo museos descargado con éxito
11:16:46 a. m.: INFO [main.py:155] Se transforma data de museos
11:16:46 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:16:47 a. m.: INFO [main.py:141] Comienza descarga de salas-de-cine
11:16:47 a. m.: INFO [main.py:144] Archivo salas-de-cine descargado con éxito
11:16:47 a. m.: INFO [main.py:158] Se transforma data de salas-de-cine
11:16:47 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:16:49 a. m.: INFO [main.py:141] Comienza descarga de bibliotecas
11:16:49 a. m.: INFO [main.py:144] Archivo bibliotecas descargado con éxito
11:16:49 a. m.: INFO [main.py:158] Se transforma data de bibliotecas
11:19:03 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:19:03 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:19:03 a. m.: INFO [main.py:186] Challenge de Data Analytics - Alkemy
11:19:03 a. m.: INFO [main.py:144] Ejecución ArchivosCsv.descarga_archivos()
11:19:03 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:19:06 a. m.: INFO [main.py:156] Comienza descarga de museos
11:19:06 a. m.: INFO [main.py:159] Archivo museos descargado con éxito
11:19:06 a. m.: INFO [main.py:170] Se transforma data de museos
11:19:06 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:19:08 a. m.: INFO [main.py:156] Comienza descarga de salas-de-cine
11:19:08 a. m.: INFO [main.py:159] Archivo salas-de-cine descargado con éxito
11:19:08 a. m.: INFO [main.py:173] Se transforma data de salas-de-cine
11:19:08 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:19:10 a. m.: INFO [main.py:156] Comienza descarga de bibliotecas
11:19:10 a. m.: INFO [main.py:159] Archivo bibliotecas descargado con éxito
11:19:10 a. m.: INFO [main.py:173] Se transforma data de bibliotecas
11:21:35 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:21:35 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:21:35 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:21:35 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:21:35 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:21:37 a. m.: INFO [main.py:164] Comienza descarga de museos
11:21:37 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:21:37 a. m.: INFO [main.py:178] Se transforma data de museos
11:21:37 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:21:39 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:21:39 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:21:39 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:21:53 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:21:53 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:21:53 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:21:53 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:21:53 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:21:54 a. m.: INFO [main.py:164] Comienza descarga de museos
11:21:54 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:21:54 a. m.: INFO [main.py:178] Se transforma data de museos
11:21:54 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:21:55 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:21:55 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:21:55 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:23:01 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:23:01 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:23:01 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:23:01 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:23:01 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:23:03 a. m.: INFO [main.py:164] Comienza descarga de museos
11:23:03 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:23:03 a. m.: INFO [main.py:178] Se transforma data de museos
11:23:03 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:23:05 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:23:05 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:23:05 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:23:26 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:23:26 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:23:26 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:23:26 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:23:26 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:23:29 a. m.: INFO [main.py:164] Comienza descarga de museos
11:23:29 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:23:29 a. m.: INFO [main.py:178] Se transforma data de museos
11:23:29 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:23:30 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:23:30 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:23:30 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:23:30 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:23:32 a. m.: INFO [main.py:164] Comienza descarga de bibliotecas
11:23:32 a. m.: INFO [main.py:167] Archivo bibliotecas descargado con éxito
11:23:32 a. m.: INFO [main.py:181] Se transforma data de bibliotecas
11:24:22 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:24:22 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:24:22 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:24:22 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:24:22 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:24:24 a. m.: INFO [main.py:164] Comienza descarga de museos
11:24:24 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:24:24 a. m.: INFO [main.py:178] Se transforma data de museos
11:24:24 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:24:25 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:24:25 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:24:25 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:24:47 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:24:47 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:24:47 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:24:47 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:24:47 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:24:49 a. m.: INFO [main.py:164] Comienza descarga de museos
11:24:49 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:24:49 a. m.: INFO [main.py:178] Se transforma data de museos
11:24:49 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:24:50 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:24:50 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:24:50 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:24:50 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:24:52 a. m.: INFO [main.py:164] Comienza descarga de bibliotecas
11:24:52 a. m.: INFO [main.py:167] Archivo bibliotecas descargado con éxito
11:24:52 a. m.: INFO [main.py:181] Se transforma data de bibliotecas
11:26:32 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:26:32 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:26:32 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:26:32 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:26:32 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:26:35 a. m.: INFO [main.py:164] Comienza descarga de museos
11:26:35 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:26:35 a. m.: INFO [main.py:178] Se transforma data de museos
11:26:35 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:26:36 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:26:36 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:26:36 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:26:36 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:26:38 a. m.: INFO [main.py:164] Comienza descarga de bibliotecas
11:26:38 a. m.: INFO [main.py:167] Archivo bibliotecas descargado con éxito
11:26:38 a. m.: INFO [main.py:181] Se transforma data de bibliotecas
11:27:04 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:27:04 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:27:04 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:27:04 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:27:04 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:27:06 a. m.: INFO [main.py:164] Comienza descarga de museos
11:27:06 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:27:06 a. m.: INFO [main.py:178] Se transforma data de museos
11:27:06 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:27:08 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:27:08 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:27:08 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:27:08 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:27:09 a. m.: INFO [main.py:164] Comienza descarga de bibliotecas
11:27:09 a. m.: INFO [main.py:167] Archivo bibliotecas descargado con éxito
11:27:09 a. m.: INFO [main.py:181] Se transforma data de bibliotecas
11:28:56 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:28:56 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:28:56 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:28:56 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:28:56 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:28:58 a. m.: INFO [main.py:164] Comienza descarga de museos
11:28:58 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:28:58 a. m.: INFO [main.py:178] Se transforma data de museos
11:28:58 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:29:00 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:29:00 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:29:00 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:29:39 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:29:39 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:29:39 a. m.: INFO [main.py:194] Challenge de Data Analytics - Alkemy
11:29:39 a. m.: INFO [main.py:152] Ejecución ArchivosCsv.descarga_archivos()
11:29:39 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:29:41 a. m.: INFO [main.py:164] Comienza descarga de museos
11:29:41 a. m.: INFO [main.py:167] Archivo museos descargado con éxito
11:29:41 a. m.: INFO [main.py:178] Se transforma data de museos
11:29:41 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:29:41 a. m.: INFO [main.py:164] Comienza descarga de salas-de-cine
11:29:41 a. m.: INFO [main.py:167] Archivo salas-de-cine descargado con éxito
11:29:41 a. m.: INFO [main.py:181] Se transforma data de salas-de-cine
11:29:41 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:29:43 a. m.: INFO [main.py:164] Comienza descarga de bibliotecas
11:29:43 a. m.: INFO [main.py:167] Archivo bibliotecas descargado con éxito
11:29:43 a. m.: INFO [main.py:181] Se transforma data de bibliotecas
11:33:21 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:33:21 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:33:21 a. m.: INFO [main.py:197] Challenge de Data Analytics - Alkemy
11:33:21 a. m.: INFO [main.py:155] Ejecución ArchivosCsv.descarga_archivos()
11:33:21 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:33:24 a. m.: INFO [main.py:167] Comienza descarga de museos
11:33:24 a. m.: INFO [main.py:170] Archivo museos descargado con éxito
11:33:24 a. m.: INFO [main.py:181] Se transforma data de museos
11:33:24 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:33:25 a. m.: INFO [main.py:167] Comienza descarga de salas-de-cine
11:33:25 a. m.: INFO [main.py:170] Archivo salas-de-cine descargado con éxito
11:33:25 a. m.: INFO [main.py:184] Se transforma data de salas-de-cine
11:33:25 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:33:27 a. m.: INFO [main.py:167] Comienza descarga de bibliotecas
11:33:27 a. m.: INFO [main.py:170] Archivo bibliotecas descargado con éxito
11:33:27 a. m.: INFO [main.py:184] Se transforma data de bibliotecas
11:37:43 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:37:43 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:37:43 a. m.: INFO [main.py:201] Challenge de Data Analytics - Alkemy
11:37:43 a. m.: INFO [main.py:159] Ejecución ArchivosCsv.descarga_archivos()
11:37:43 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:37:45 a. m.: INFO [main.py:171] Comienza descarga de museos
11:37:46 a. m.: INFO [main.py:174] Archivo museos descargado con éxito
11:37:46 a. m.: INFO [main.py:185] Se transforma data de museos
11:37:46 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:37:47 a. m.: INFO [main.py:171] Comienza descarga de salas-de-cine
11:37:47 a. m.: INFO [main.py:174] Archivo salas-de-cine descargado con éxito
11:37:47 a. m.: INFO [main.py:188] Se transforma data de salas-de-cine
11:38:17 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:38:17 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:38:17 a. m.: INFO [main.py:199] Challenge de Data Analytics - Alkemy
11:38:17 a. m.: INFO [main.py:157] Ejecución ArchivosCsv.descarga_archivos()
11:38:17 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:38:19 a. m.: INFO [main.py:169] Comienza descarga de museos
11:38:19 a. m.: INFO [main.py:172] Archivo museos descargado con éxito
11:38:19 a. m.: INFO [main.py:183] Se transforma data de museos
11:38:19 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:38:20 a. m.: INFO [main.py:169] Comienza descarga de salas-de-cine
11:38:20 a. m.: INFO [main.py:172] Archivo salas-de-cine descargado con éxito
11:38:20 a. m.: INFO [main.py:186] Se transforma data de salas-de-cine
11:38:20 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:38:22 a. m.: INFO [main.py:169] Comienza descarga de bibliotecas
11:38:22 a. m.: INFO [main.py:172] Archivo bibliotecas descargado con éxito
11:38:22 a. m.: INFO [main.py:186] Se transforma data de bibliotecas
11:39:46 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:39:46 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:39:46 a. m.: INFO [main.py:199] Challenge de Data Analytics - Alkemy
11:39:46 a. m.: INFO [main.py:157] Ejecución ArchivosCsv.descarga_archivos()
11:39:46 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:39:48 a. m.: INFO [main.py:169] Comienza descarga de museos
11:39:48 a. m.: INFO [main.py:172] Archivo museos descargado con éxito
11:39:48 a. m.: INFO [main.py:183] Se transforma data de museos
11:39:48 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:39:49 a. m.: INFO [main.py:169] Comienza descarga de salas-de-cine
11:39:49 a. m.: INFO [main.py:172] Archivo salas-de-cine descargado con éxito
11:39:49 a. m.: INFO [main.py:186] Se transforma data de salas-de-cine
11:39:49 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:39:51 a. m.: INFO [main.py:169] Comienza descarga de bibliotecas
11:39:51 a. m.: INFO [main.py:172] Archivo bibliotecas descargado con éxito
11:39:51 a. m.: INFO [main.py:186] Se transforma data de bibliotecas
11:40:59 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:40:59 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:40:59 a. m.: INFO [main.py:201] Challenge de Data Analytics - Alkemy
11:40:59 a. m.: INFO [main.py:159] Ejecución ArchivosCsv.descarga_archivos()
11:40:59 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:01 a. m.: INFO [main.py:171] Comienza descarga de museos
11:41:01 a. m.: INFO [main.py:174] Archivo museos descargado con éxito
11:41:01 a. m.: INFO [main.py:185] Se transforma data de museos
11:41:01 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:03 a. m.: INFO [main.py:171] Comienza descarga de salas-de-cine
11:41:03 a. m.: INFO [main.py:174] Archivo salas-de-cine descargado con éxito
11:41:03 a. m.: INFO [main.py:188] Se transforma data de salas-de-cine
11:41:03 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:04 a. m.: INFO [main.py:171] Comienza descarga de bibliotecas
11:41:05 a. m.: INFO [main.py:174] Archivo bibliotecas descargado con éxito
11:41:05 a. m.: INFO [main.py:188] Se transforma data de bibliotecas
11:41:11 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:41:11 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:41:11 a. m.: INFO [main.py:201] Challenge de Data Analytics - Alkemy
11:41:11 a. m.: INFO [main.py:159] Ejecución ArchivosCsv.descarga_archivos()
11:41:11 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:13 a. m.: INFO [main.py:171] Comienza descarga de museos
11:41:13 a. m.: INFO [main.py:174] Archivo museos descargado con éxito
11:41:13 a. m.: INFO [main.py:185] Se transforma data de museos
11:41:13 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:14 a. m.: INFO [main.py:171] Comienza descarga de salas-de-cine
11:41:14 a. m.: INFO [main.py:174] Archivo salas-de-cine descargado con éxito
11:41:14 a. m.: INFO [main.py:188] Se transforma data de salas-de-cine
11:41:14 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:16 a. m.: INFO [main.py:171] Comienza descarga de bibliotecas
11:41:16 a. m.: INFO [main.py:174] Archivo bibliotecas descargado con éxito
11:41:16 a. m.: INFO [main.py:188] Se transforma data de bibliotecas
11:41:26 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:41:26 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:41:26 a. m.: INFO [main.py:201] Challenge de Data Analytics - Alkemy
11:41:26 a. m.: INFO [main.py:159] Ejecución ArchivosCsv.descarga_archivos()
11:41:26 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:28 a. m.: INFO [main.py:171] Comienza descarga de museos
11:41:28 a. m.: INFO [main.py:174] Archivo museos descargado con éxito
11:41:28 a. m.: INFO [main.py:185] Se transforma data de museos
11:41:28 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:28 a. m.: INFO [main.py:171] Comienza descarga de salas-de-cine
11:41:28 a. m.: INFO [main.py:174] Archivo salas-de-cine descargado con éxito
11:41:28 a. m.: INFO [main.py:188] Se transforma data de salas-de-cine
11:41:29 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:41:30 a. m.: INFO [main.py:171] Comienza descarga de bibliotecas
11:41:30 a. m.: INFO [main.py:174] Archivo bibliotecas descargado con éxito
11:41:30 a. m.: INFO [main.py:188] Se transforma data de bibliotecas
11:42:04 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:42:04 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:42:04 a. m.: INFO [main.py:201] Challenge de Data Analytics - Alkemy
11:42:04 a. m.: INFO [main.py:159] Ejecución ArchivosCsv.descarga_archivos()
11:42:04 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:42:06 a. m.: INFO [main.py:171] Comienza descarga de museos
11:42:06 a. m.: INFO [main.py:174] Archivo museos descargado con éxito
11:42:06 a. m.: INFO [main.py:185] Se transforma data de museos
11:42:06 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:42:07 a. m.: INFO [main.py:171] Comienza descarga de salas-de-cine
11:42:07 a. m.: INFO [main.py:174] Archivo salas-de-cine descargado con éxito
11:42:07 a. m.: INFO [main.py:188] Se transforma data de salas-de-cine
11:42:07 a. m.: INFO [main.py:54] Directorios ya existentes. Continúa el proceso de descarga.
11:42:09 a. m.: INFO [main.py:171] Comienza descarga de bibliotecas
11:42:09 a. m.: INFO [main.py:174] Archivo bibliotecas descargado con éxito
11:42:09 a. m.: INFO [main.py:188] Se transforma data de bibliotecas
11:44:45 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:44:45 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:44:45 a. m.: INFO [main.py:182] Challenge de Data Analytics - Alkemy
11:44:45 a. m.: INFO [main.py:140] Ejecución ArchivosCsv.descarga_archivos()
11:44:45 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:44:47 a. m.: INFO [main.py:152] Comienza descarga de museos
11:44:47 a. m.: INFO [main.py:155] Archivo museos descargado con éxito
11:44:47 a. m.: INFO [main.py:166] Se transforma data de museos
11:44:47 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:44:48 a. m.: INFO [main.py:152] Comienza descarga de salas-de-cine
11:44:48 a. m.: INFO [main.py:155] Archivo salas-de-cine descargado con éxito
11:44:48 a. m.: INFO [main.py:169] Se transforma data de salas-de-cine
11:46:11 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:46:11 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:46:11 a. m.: INFO [main.py:182] Challenge de Data Analytics - Alkemy
11:46:11 a. m.: INFO [main.py:140] Ejecución ArchivosCsv.descarga_archivos()
11:46:11 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:46:13 a. m.: INFO [main.py:152] Comienza descarga de museos
11:46:13 a. m.: INFO [main.py:155] Archivo museos descargado con éxito
11:46:13 a. m.: INFO [main.py:166] Se transforma data de museos
11:46:13 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:46:15 a. m.: INFO [main.py:152] Comienza descarga de salas-de-cine
11:46:15 a. m.: INFO [main.py:155] Archivo salas-de-cine descargado con éxito
11:46:15 a. m.: INFO [main.py:169] Se transforma data de salas-de-cine
11:46:29 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:46:29 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:46:29 a. m.: INFO [main.py:182] Challenge de Data Analytics - Alkemy
11:46:29 a. m.: INFO [main.py:140] Ejecución ArchivosCsv.descarga_archivos()
11:46:29 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:46:31 a. m.: INFO [main.py:152] Comienza descarga de museos
11:46:31 a. m.: INFO [main.py:155] Archivo museos descargado con éxito
11:46:31 a. m.: INFO [main.py:166] Se transforma data de museos
11:46:31 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:46:32 a. m.: INFO [main.py:152] Comienza descarga de salas-de-cine
11:46:32 a. m.: INFO [main.py:155] Archivo salas-de-cine descargado con éxito
11:46:32 a. m.: INFO [main.py:169] Se transforma data de salas-de-cine
11:46:32 a. m.: INFO [main.py:53] Directorios ya existentes. Continúa el proceso de descarga.
11:46:34 a. m.: INFO [main.py:152] Comienza descarga de bibliotecas
11:46:34 a. m.: INFO [main.py:155] Archivo bibliotecas descargado con éxito
11:46:34 a. m.: INFO [main.py:169] Se transforma data de bibliotecas
11:51:33 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:51:33 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:51:33 a. m.: INFO [main.py:166] Challenge de Data Analytics - Alkemy
11:51:33 a. m.: INFO [main.py:124] Ejecución ArchivosCsv.descarga_archivos()
11:51:33 a. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
11:51:36 a. m.: INFO [main.py:136] Comienza descarga de museos
11:51:36 a. m.: INFO [main.py:139] Archivo museos descargado con éxito
11:51:36 a. m.: INFO [main.py:150] Se transforma data de museos
11:51:36 a. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
11:51:37 a. m.: INFO [main.py:136] Comienza descarga de salas-de-cine
11:51:37 a. m.: INFO [main.py:139] Archivo salas-de-cine descargado con éxito
11:51:37 a. m.: INFO [main.py:153] Se transforma data de salas-de-cine
11:51:37 a. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
11:51:39 a. m.: INFO [main.py:136] Comienza descarga de bibliotecas
11:51:39 a. m.: INFO [main.py:139] Archivo bibliotecas descargado con éxito
11:51:39 a. m.: INFO [main.py:153] Se transforma data de bibliotecas
11:56:03 a. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
11:56:03 a. m.: INFO [postgres_client.py:42] Engine creado con éxito
11:56:03 a. m.: INFO [main.py:163] Challenge de Data Analytics - Alkemy
11:56:03 a. m.: INFO [main.py:121] Ejecución ArchivosCsv.descarga_archivos()
11:56:03 a. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
11:56:05 a. m.: INFO [main.py:133] Comienza descarga de museos
11:56:05 a. m.: INFO [main.py:136] Archivo museos descargado con éxito
11:56:05 a. m.: INFO [main.py:147] Se transforma data de museos
11:56:05 a. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
11:56:07 a. m.: INFO [main.py:133] Comienza descarga de salas-de-cine
11:56:07 a. m.: INFO [main.py:136] Archivo salas-de-cine descargado con éxito
11:56:07 a. m.: INFO [main.py:150] Se transforma data de salas-de-cine
11:56:07 a. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
11:56:09 a. m.: INFO [main.py:133] Comienza descarga de bibliotecas
11:56:09 a. m.: INFO [main.py:136] Archivo bibliotecas descargado con éxito
11:56:09 a. m.: INFO [main.py:150] Se transforma data de bibliotecas
12:19:32 p. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
12:19:32 p. m.: INFO [postgres_client.py:42] Engine creado con éxito
12:19:32 p. m.: INFO [main.py:164] Challenge de Data Analytics - Alkemy
12:19:32 p. m.: INFO [main.py:120] Ejecución ArchivosCsv.descarga_archivos()
12:19:32 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:19:35 p. m.: INFO [main.py:132] Comienza descarga de museos
12:19:35 p. m.: INFO [main.py:135] Archivo museos descargado con éxito
12:19:35 p. m.: INFO [main.py:146] Se transforma data de museos
12:19:35 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:19:36 p. m.: INFO [main.py:132] Comienza descarga de salas-de-cine
12:19:36 p. m.: INFO [main.py:135] Archivo salas-de-cine descargado con éxito
12:19:36 p. m.: INFO [main.py:149] Se transforma data de salas-de-cine
12:19:36 p. m.: INFO [postgres_client.py:64] ---------------------------------------data_cines_to_postgres---------------------------------------
12:19:36 p. m.: INFO [postgres_client.py:83] Transformando data
12:19:36 p. m.: INFO [postgres_client.py:86] Datos de "salas de cine" cargados con éxitos
12:19:36 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:19:38 p. m.: INFO [main.py:132] Comienza descarga de bibliotecas
12:19:38 p. m.: INFO [main.py:135] Archivo bibliotecas descargado con éxito
12:19:38 p. m.: INFO [main.py:149] Se transforma data de bibliotecas
12:19:38 p. m.: INFO [postgres_client.py:47] Cargando información consolidada a la base de datos
12:20:25 p. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
12:20:25 p. m.: INFO [postgres_client.py:42] Engine creado con éxito
12:20:25 p. m.: INFO [main.py:164] Challenge de Data Analytics - Alkemy
12:20:25 p. m.: INFO [main.py:120] Ejecución ArchivosCsv.descarga_archivos()
12:20:25 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:20:26 p. m.: INFO [main.py:132] Comienza descarga de museos
12:20:26 p. m.: INFO [main.py:135] Archivo museos descargado con éxito
12:20:26 p. m.: INFO [main.py:146] Se transforma data de museos
12:20:26 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:20:27 p. m.: INFO [main.py:132] Comienza descarga de salas-de-cine
12:20:27 p. m.: INFO [main.py:135] Archivo salas-de-cine descargado con éxito
12:20:27 p. m.: INFO [main.py:149] Se transforma data de salas-de-cine
12:20:27 p. m.: INFO [postgres_client.py:62] ---------------------------------------data_cines_to_postgres---------------------------------------
12:20:27 p. m.: INFO [postgres_client.py:81] Transformando data
12:20:27 p. m.: INFO [postgres_client.py:84] Datos de "salas de cine" cargados con éxitos
12:20:27 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:20:29 p. m.: INFO [main.py:132] Comienza descarga de bibliotecas
12:20:29 p. m.: INFO [main.py:135] Archivo bibliotecas descargado con éxito
12:20:29 p. m.: INFO [main.py:149] Se transforma data de bibliotecas
12:20:29 p. m.: INFO [postgres_client.py:47] Cargando información consolidada a la base de datos
12:20:29 p. m.: INFO [postgres_client.py:54] Transformando data
12:20:29 p. m.: INFO [postgres_client.py:57] Información consolidada cargada con éxito
12:22:15 p. m.: INFO [postgres_client.py:41] Conectado con éxito a data_analytics
12:22:15 p. m.: INFO [postgres_client.py:42] Engine creado con éxito
12:22:15 p. m.: INFO [main.py:164] Challenge de Data Analytics - Alkemy
12:22:15 p. m.: INFO [main.py:120] Ejecución ArchivosCsv.descarga_archivos()
12:22:15 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:22:17 p. m.: INFO [main.py:132] Comienza descarga de museos
12:22:17 p. m.: INFO [main.py:135] Archivo museos descargado con éxito
12:22:17 p. m.: INFO [main.py:146] Se transforma data de museos
12:22:17 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:22:18 p. m.: INFO [main.py:132] Comienza descarga de salas-de-cine
12:22:18 p. m.: INFO [main.py:135] Archivo salas-de-cine descargado con éxito
12:22:18 p. m.: INFO [main.py:149] Se transforma data de salas-de-cine
12:22:18 p. m.: INFO [postgres_client.py:63] ---------------------------------------data_cines_to_postgres---------------------------------------
12:22:18 p. m.: INFO [postgres_client.py:82] Transformando data
12:22:19 p. m.: INFO [postgres_client.py:85] Datos de "salas de cine" cargados con éxitos
12:22:19 p. m.: INFO [main.py:45] Directorios ya existentes. Continúa el proceso de descarga.
12:22:20 p. m.: INFO [main.py:132] Comienza descarga de bibliotecas
12:22:20 p. m.: INFO [main.py:135] Archivo bibliotecas descargado con éxito
12:22:20 p. m.: INFO [main.py:149] Se transforma data de bibliotecas
12:22:20 p. m.: INFO [postgres_client.py:47] Cargando información consolidada a la base de datos
12:22:20 p. m.: INFO [postgres_client.py:55] Transformando data
12:22:21 p. m.: INFO [postgres_client.py:58] Información consolidada cargada con éxito
09:01:28 PM: INFO [postgres_client.py:31] Conectado con éxito a data_analytics
09:01:28 PM: INFO [postgres_client.py:32] Engine creado con éxito
09:01:28 p. m.: INFO [main.py:190] Challenge de Data Analytics - Alkemy
09:01:28 p. m.: INFO [main.py:104] Ejecución ArchivosCsv.descarga_archivos()
09:01:28 p. m.: INFO [main.py:47] Directorios ya existentes. Continúa el proceso de descarga.
09:01:30 p. m.: INFO [main.py:116] Comienza descarga de museos
09:01:30 p. m.: INFO [main.py:118] Archivo museos descargado con éxito
09:01:30 p. m.: INFO [main.py:127] Comienza la extracción de los datos
09:01:30 p. m.: INFO [main.py:129] Se extrae data de museos
09:01:31 p. m.: INFO [main.py:47] Directorios ya existentes. Continúa el proceso de descarga.
09:01:32 p. m.: INFO [main.py:116] Comienza descarga de salas-de-cine
09:01:32 p. m.: INFO [main.py:118] Archivo salas-de-cine descargado con éxito
09:01:32 p. m.: INFO [main.py:127] Comienza la extracción de los datos
09:01:32 p. m.: INFO [main.py:133] Se extrae data de salas-de-cine
09:01:32 p. m.: INFO [postgres_client.py:121] ---------------------------------------data_cines_to_postgres---------------------------------------
09:01:32 p. m.: INFO [postgres_client.py:140] Cargando data
09:01:32 p. m.: INFO [postgres_client.py:143] Datos de "salas de cine" cargados con éxitos
09:01:32 p. m.: INFO [main.py:47] Directorios ya existentes. Continúa el proceso de descarga.
09:01:35 p. m.: INFO [main.py:116] Comienza descarga de bibliotecas
09:01:35 p. m.: INFO [main.py:118] Archivo bibliotecas descargado con éxito
09:01:35 p. m.: INFO [main.py:127] Comienza la extracción de los datos
09:01:35 p. m.: INFO [main.py:133] Se extrae data de bibliotecas
09:01:35 p. m.: INFO [postgres_client.py:37] Cargando información consolidada a la base de datos
09:01:35 p. m.: INFO [postgres_client.py:50] Transformando data_consolidada
09:01:35 p. m.: INFO [postgres_client.py:53] Información consolidada cargada con éxito
09:01:35 p. m.: INFO [postgres_client.py:58] -------------------------------Carga de datos a tablas totalizadoras--------------------------------
09:01:35 p. m.: INFO [postgres_client.py:90] Comienza la carga de datos
09:01:35 p. m.: INFO [postgres_client.py:94] Información insertada a totales_provincia_y_cat con éxito
09:01:35 p. m.: INFO [postgres_client.py:98] Toda la información fue insertada con éxito
09:01:35 p. m.: INFO [postgres_client.py:94] Información insertada a totales_categoria con éxito
09:01:35 p. m.: INFO [postgres_client.py:98] Toda la información fue insertada con éxito
09:01:35 p. m.: INFO [postgres_client.py:94] Información insertada a totales_fuente con éxito
09:01:35 p. m.: INFO [postgres_client.py:98] Toda la información fue insertada con éxito
09:19:15 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:19:15 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:19:15 p. m.: INFO [main.py:190] Challenge de Data Analytics - Alkemy
09:19:15 p. m.: INFO [main.py:107] Ejecución ArchivosCsv.descarga_archivos()
09:19:15 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:19:18 p. m.: INFO [main.py:119] Comienza descarga de museos
09:19:18 p. m.: INFO [main.py:121] Archivo museos descargado con éxito
09:19:18 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:19:18 p. m.: INFO [main.py:132] Se extrae data de museos
09:19:18 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:21:16 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:21:16 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:21:28 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:21:28 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:21:32 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:21:32 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:21:35 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:21:35 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:21:59 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:21:59 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:23:01 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:23:01 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:23:01 p. m.: INFO [main.py:190] Challenge de Data Analytics - Alkemy
09:23:01 p. m.: INFO [main.py:107] Ejecución ArchivosCsv.descarga_archivos()
09:23:01 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:23:03 p. m.: INFO [main.py:119] Comienza descarga de museos
09:23:03 p. m.: INFO [main.py:121] Archivo museos descargado con éxito
09:23:03 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:23:03 p. m.: INFO [main.py:132] Se extrae data de museos
09:23:03 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:23:05 p. m.: INFO [main.py:119] Comienza descarga de salas-de-cine
09:23:05 p. m.: INFO [main.py:121] Archivo salas-de-cine descargado con éxito
09:23:05 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:23:05 p. m.: INFO [main.py:136] Se extrae data de salas-de-cine
09:23:05 p. m.: INFO [postgres_client.py:120] data_cines_to_postgres
09:23:05 p. m.: INFO [postgres_client.py:139] Cargando data
09:23:05 p. m.: INFO [postgres_client.py:142] Datos de "salas de cine" cargados con éxitos
09:23:05 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:23:07 p. m.: INFO [main.py:119] Comienza descarga de bibliotecas
09:23:07 p. m.: INFO [main.py:121] Archivo bibliotecas descargado con éxito
09:23:07 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:23:07 p. m.: INFO [main.py:136] Se extrae data de bibliotecas
09:23:07 p. m.: INFO [postgres_client.py:47] Cargando información consolidada a la base de datos
09:23:07 p. m.: INFO [postgres_client.py:57] Carga data_consolidada
09:23:08 p. m.: INFO [postgres_client.py:60] Información consolidada cargada con éxito
09:23:08 p. m.: INFO [postgres_client.py:71] Carga de datos a tablas totalizadoras
09:23:08 p. m.: INFO [postgres_client.py:102] Comienza la carga de datos
09:23:08 p. m.: INFO [postgres_client.py:106] Información insertada a totales_provincia_y_cat con éxito
09:23:08 p. m.: INFO [postgres_client.py:110] Toda la información fue insertada con éxito
09:23:08 p. m.: INFO [postgres_client.py:106] Información insertada a totales_categoria con éxito
09:23:08 p. m.: INFO [postgres_client.py:110] Toda la información fue insertada con éxito
09:23:08 p. m.: INFO [postgres_client.py:106] Información insertada a totales_fuente con éxito
09:23:08 p. m.: INFO [postgres_client.py:110] Toda la información fue insertada con éxito
09:23:19 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:23:19 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:23:19 p. m.: INFO [main.py:190] Challenge de Data Analytics - Alkemy
09:23:19 p. m.: INFO [main.py:107] Ejecución ArchivosCsv.descarga_archivos()
09:23:19 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:23:21 p. m.: INFO [main.py:119] Comienza descarga de museos
09:23:21 p. m.: INFO [main.py:121] Archivo museos descargado con éxito
09:23:21 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:23:21 p. m.: INFO [main.py:132] Se extrae data de museos
09:23:21 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:23:22 p. m.: INFO [main.py:119] Comienza descarga de salas-de-cine
09:23:22 p. m.: INFO [main.py:121] Archivo salas-de-cine descargado con éxito
09:23:22 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:23:22 p. m.: INFO [main.py:136] Se extrae data de salas-de-cine
09:23:22 p. m.: INFO [postgres_client.py:120] data_cines_to_postgres
09:23:22 p. m.: INFO [postgres_client.py:139] Cargando data
09:23:22 p. m.: INFO [postgres_client.py:142] Datos de "salas de cine" cargados con éxitos
09:23:22 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:23:24 p. m.: INFO [main.py:119] Comienza descarga de bibliotecas
09:23:24 p. m.: INFO [main.py:121] Archivo bibliotecas descargado con éxito
09:23:24 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:23:24 p. m.: INFO [main.py:136] Se extrae data de bibliotecas
09:23:24 p. m.: INFO [postgres_client.py:47] Cargando información consolidada a la base de datos
09:23:24 p. m.: INFO [postgres_client.py:57] Carga data_consolidada
09:23:24 p. m.: INFO [postgres_client.py:60] Información consolidada cargada con éxito
09:24:28 PM: INFO [postgres_client.py:34] Engine creado con éxito
09:24:28 PM: INFO [postgres_client.py:35] Conectado con éxito a data_analytics
09:24:28 p. m.: INFO [main.py:190] Challenge de Data Analytics - Alkemy
09:24:28 p. m.: INFO [main.py:107] Ejecución ArchivosCsv.descarga_archivos()
09:24:28 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:24:30 p. m.: INFO [main.py:119] Comienza descarga de museos
09:24:30 p. m.: INFO [main.py:121] Archivo museos descargado con éxito
09:24:30 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:24:30 p. m.: INFO [main.py:132] Se extrae data de museos
09:24:30 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:24:32 p. m.: INFO [main.py:119] Comienza descarga de salas-de-cine
09:24:32 p. m.: INFO [main.py:121] Archivo salas-de-cine descargado con éxito
09:24:32 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:24:32 p. m.: INFO [main.py:136] Se extrae data de salas-de-cine
09:24:32 p. m.: INFO [postgres_client.py:120] data_cines_to_postgres
09:24:32 p. m.: INFO [postgres_client.py:139] Cargando data
09:24:32 p. m.: INFO [postgres_client.py:142] Datos de "salas de cine" cargados con éxitos
09:24:32 p. m.: INFO [main.py:50] Directorios ya existentes. Continúa el proceso de descarga.
09:24:35 p. m.: INFO [main.py:119] Comienza descarga de bibliotecas
09:24:35 p. m.: INFO [main.py:121] Archivo bibliotecas descargado con éxito
09:24:35 p. m.: INFO [main.py:130] Comienza la extracción de los datos
09:24:35 p. m.: INFO [main.py:136] Se extrae data de bibliotecas
09:24:35 p. m.: INFO [postgres_client.py:47] Cargando información consolidada a la base de datos
09:24:35 p. m.: INFO [postgres_client.py:57] Carga data_consolidada
09:24:35 p. m.: INFO [postgres_client.py:60] Información consolidada cargada con éxito
09:24:35 p. m.: INFO [postgres_client.py:71] Carga de datos a tablas totalizadoras
09:24:35 p. m.: INFO [postgres_client.py:102] Comienza la carga de datos
09:24:35 p. m.: INFO [postgres_client.py:106] Información insertada a totales_provincia_y_cat con éxito
09:24:35 p. m.: INFO [postgres_client.py:110] Toda la información fue insertada con éxito
09:24:35 p. m.: INFO [postgres_client.py:106] Información insertada a totales_categoria con éxito
09:24:35 p. m.: INFO [postgres_client.py:110] Toda la información fue insertada con éxito
09:24:35 p. m.: INFO [postgres_client.py:106] Información insertada a totales_fuente con éxito
09:24:35 p. m.: INFO [postgres_client.py:110] Toda la información fue insertada con éxito
